### PR TITLE
Added Node and MCO features to RN

### PR DESCRIPTION
**Pull request by mburke5678**
Add to RN:
https://issues.redhat.com/browse/OSDOCS-1644
https://issues.redhat.com/browse/OSDOCS-1645

### CODE REVIEW

**Comment by mburke5678** [comment]
Preview: https://deploy-preview-29615--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-machine-api-avoid-reboots
https://deploy-preview-29615--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-nodes-using-image-short-names-containerRuntimeSearchRegistries

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -542,6 +542,19 @@ For more information, see xref:../machine_management/creating_machinesets/creati
...
+
+The Machine Config Operator (MCO) watches your cluster for configuration changes to nodes. When it detects these changes, it drains and reboots the nodes. The MCO now no longer reboots nodes after these specific changes:
+
+* changes to the SSH key in the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter of a machine config
+* changes to the global pull secret or pull secret in the `openshift-config` namespace
+* changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object
+ 
+When the MCO detects these changes, it drains the corresponding nodes, applies the changes, and uncordons the nodes, but does not reboot the nodes.
+
+For more information, see xref:../architecture/control-plane.html#understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:556]: s/.html/.adoc

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -591,6 +604,13 @@ You can now configure a priority class to be non-preempting by setting the `pree
 
 For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preempting-priority-class_nodes-pods-priority[Non-preempting priority classes].
 
+[id="ocp-4-7-nodes-using-image-short-names-containerRuntimeSearchRegistries"]
+==== Enabling image registries that allow image short names
+
+You can now add registries that allow searching for an image by using an image short name. An image short name enables you to search for images without including the fully qualified domain name in the pull spec. Using image short names with public registries is strongly discouraged. You should use image short names with only internal or private registries.
+
+For more information, see xref:../openshift_images/image-configuration.html#images-configuration-shortname_image-configuration[Adding registries that allow image short names].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:612]: s/.html/.adoc

**Comment by ahardin-rh** [comment]
Just 2 comments. Otherwise, looks great!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (mburke5678) [reply release_notes/ocp-4-7-release-notes.adoc]: I always do that. Thanks!

**Comment by mburke5678** [comment]
Closing as a dupe of https://github.com/openshift/openshift-docs/commit/1b8da7440470f7634d49de4dbbfda3715c9f804a#diff-14b6355bc5db567351b9e43a469c73396794e644c2dd1fc8523417f2f7b72c16R583

**Pull request closed without merging**